### PR TITLE
EPMRPP-78105 || Notifications preloader, EPMRPP-78115 || Modal window fix

### PR DESCRIPTION
**Pull request by tr1ble**
EPMRPP-78105 || Notifications preloader, EPMRPP-78115 || Modal window paddings fix

### CODE REVIEW

**Review by chivekrodis — CHANGES_REQUESTED** [review]

File: app/src/componentLibrary/modal/modalLayout/modalLayout.scss [hunk 1]
```diff
@@ -75,7 +75,7 @@ $WIDTH-LARGE: 720px;
   display: inline-block;
   margin-top: 10px;
   margin-bottom: 10px;
-  padding: 32px 48px;
+  padding: 32px 40px;
```
[the remark this answers is not in the document]
Comment (tr1ble) [reply app/src/componentLibrary/modal/modalLayout/modalLayout.scss:78]: It's the task to fix it in new modal window so yes I guess that it's discussed

[the remark this answers is not in the document]
Comment (chivekrodis) [reply app/src/componentLibrary/modal/modalLayout/modalLayout.scss:78]: Please discuss it with UX to make sure is everything is as expected

**Review by chivekrodis — CHANGES_REQUESTED** [review]

File: app/src/componentLibrary/modal/modalLayout/modalLayout.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (chivekrodis) [reply app/src/componentLibrary/modal/modalLayout/modalLayout.scss:78]: Please discuss it with UX to make sure is everything is as expected

File: app/src/controllers/project/sagas.js [hunk 2]
```diff
@@ -179,6 +180,7 @@ function* watchUpdateProjectNotificationsConfig() {
 }
 
 function* fetchProjectNotifications() {
+  yield put(setProjectNotificationsLoadingAction(true));
   const projectId = yield select(projectIdSelector);
   const notifications = yield call(fetch, URLS.notification(projectId));
   yield put(fetchProjectNotificationsSuccessAction(notifications));
```
[the remark this answers is not in the document]
Comment (AmsterGet) [reply app/src/controllers/project/sagas.js:186]: May be `fetchDataAction` together with `fetchReducer` & `loadingReducer` can be used to cover all corner cases?

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/componentLibrary/modal/modalLayout/modalLayout.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (tr1ble) [reply app/src/componentLibrary/modal/modalLayout/modalLayout.scss:78]: Yes, I've asked, everything is fine

**Pull request merged**